src/yaws_sendfile.erl

```diff
@@ -152,7 +152,10 @@ init([]) ->
     case erl_ddll:load_driver(Dir, Shlib) of
         ok -> ok;
         {error, already_loaded} -> ok;
-        _ -> exit({error, "could not load driver " ++ Shlib})
+        {error, Reason} ->
+	    exit({error,
+		  "could not load driver " ++ Shlib,
+		  erl_ddll:format_error(Reason)})
```

Comment:
Instead of creating a 3-tuple, like this:

    {error, "could not load driver foo", "some reason string"}

let's instead keep a 2-tuple. Please amend your commit to combine the return value of `erl_ddll:format_error/1` with the value of the previous line so we get this:

    {error, "could not load driver foo: some reason string"}

Using `++` is perfectly fine here:

    exit({error,
          "could not load driver " ++ Shlib ++
          erl_ddll:format_error(Reason)}).